app/src/main/java/com/spoony/spoony/presentation/placeDetail/PlaceDetailRoute.kt

```diff
@@ -131,6 +136,17 @@ private fun PlaceDetailScreen(
 ) {
     val scrollState = rememberScrollState()
 
+    if (scoopDialogVisibility) {
+        ScoopDialog(
+            onClickPositive = {
+                onScoopButtonClick()
+            },
```

Comment:
```suggestion
            onClickPositive = onScoopButtonClick
```
P2: 이렇게도 가능할 것 같네요!!

Reply:
수정했습니다!